<?xml version='1.0'?>
<launch>
  <group ns="group1">
    <node pkg="multi_uav" type="formation_creating" name="formation_creating1" args="-x 0 -y 0 -z 0.08" output="screen" >
      <param name="uav_id" type="string" value = "1" />
    </node>
  </group>

 <group ns="group2">
    <node pkg="multi_uav" type="formation_creating" name="formation_creating2" args="-x 0 -y 3 -z 0.08" output="screen" >
      <param name="uav_id" type="string" value = "2" />
    </node>
  </group>

  <group ns="group3">
    <node pkg="multi_uav" type="formation_creating" name="formation_creating3" args="-x 2 -y 0 -z 0.08" output="screen" >
      <param name="uav_id" type="string" value = "3" />
    </node>
  </group>  

</launch>
